File: ezSCORES/ezSCORES.Model/Players.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class Players
	{
		public int Id { get; set; }

		public string FirstName { get; set; } = null!;

		public string LastName { get; set; } = null!;

		public byte[]? Picture { get; set; }

		public DateTime BirthDate { get; set; }
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/players.dart
import 'package:json_annotation/json_annotation.dart';

part 'players.g.dart';

@JsonSerializable()
class Players {
	int? id;
	String? firstName;
	String? lastName;
	String? picture;
	DateTime? birthDate;

	Players();

	factory Players.fromJson(Map<String, dynamic> json) => _$PlayersFromJson(json);

	Map<String, dynamic> toJson() => _$PlayersToJson(this);
}